packages/ui/src/bounty/queries/bounty.graphql

```diff
@@ -0,0 +1,47 @@
+fragment BountyFields on Bounty {
+  id
+  createdAt
+  title
+  cherry
+  entrantStake
+  creator {
+    ...MemberFields
+  }
+  oracle {
+    ...MemberFields
+  }
+  fundingType {
+    ... on BountyFundingLimited {
+      minFundingAmount
+      maxFundingAmount
+      fundingPeriod
+    }
+    ... on BountyFundingPerpetual {
+      target
+    }
+  }
+  workPeriod
+  judgingPeriod
+  stage
+  totalFunding
+  bountyentrybounty {
```

Comment:
This field has been renamed with PR https://github.com/Joystream/pioneer/pull/2043
```suggestion
  entries {
```